netcore/System.Private.CoreLib/shared/System.Private.CoreLib.Shared.projitems

```diff
@@ -1076,7 +1076,7 @@
     <Compile Include="$(MSBuildThisFileDirectory)Interop\Windows\Kernel32\Interop.GetSystemTime.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Interop\Windows\Kernel32\Interop.GetSystemTimeAsFileTime.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Interop\Windows\Kernel32\Interop.GetSystemTimePreciseAsFileTime.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)Interop\Windows\Kernel32\Interop.GetSystemTimes.cs" Condition="'$(FeaturePerfTracing)' == 'true'" />
+    <Compile Include="$(MSBuildThisFileDirectory)Interop\Windows\Kernel32\Interop.GetSystemTimes.cs" Condition="'$(FeaturePerfTracing)' == 'true' AND '$(FeaturePortableThreadPool)' == 'false'" />
```

Comment:
Shouldn't this be 'OR' ?

Reply:
So no FeaturePerfTracing with FeaturePortableThreadPool?

Reply:
Oh, I misread it. :)